erpnext/hr/doctype/expense_claim/expense_claim.js

```diff
@@ -152,10 +172,30 @@ erpnext.expense_claim = {
 };
 
 frappe.ui.form.on("Expense Claim", {
+	refresh: function(frm) {
+		var doc = frm.doc;
+		if(doc.advance_account)
+			frappe.call({
+				method: "erpnext.hr.doctype.expense_claim.expense_claim.update_advance_paid",
+				args: {
+					"docname": cur_frm.doc.name,
+					"employee": doc.employee,
+					"advance_account": doc.advance_account
+				},
+				callback: function(r) {
+					cur_frm.set_value("total_advance_paid", r.message.amt);
```

Comment:
Hi @mbauskar 

Thanks for the helpful review. Not sure how to add Test Cases. Could you give some guidance on this?

Kind regards,